test/e2e_tests/specs/CriticalFlow/messagesInChannels-TC-8753.spec.ts

```diff
@@ -0,0 +1,204 @@
+        await loginUser(userB, userBPageManager);
+        await userBModals.dataShareConsent().clickDecline();
+      };
+
+      await Promise.all([setupOwner(), setupMember()]);
+    });
+
+    await test.step('User A mentions User B in the channel', async () => {
+      await userAPages.conversationList().openConversation(channelName);
+      await userAPages.conversation().sendMessageWithUserMention(userB.fullName, messageText);
+    });
+
+    await test.step('User B should receive mention', async () => {
+      expect(await userBPages.conversationList().doesConversationHasMentionIndicator(channelName)).toBeTruthy();
+
+      // TODO: Bug [WPB-18226], remove this when fixed
+      await userBPageManager.refreshPage({waitUntil: 'load'});
+
+      await userBPages.conversationList().openConversation(channelName);
+      expect(await userBPages.conversation().isMessageVisible(`@${userB.fullName} ${messageText}`)).toBeTruthy();
+    });
+
+    await test.step('User A sends image', async () => {
+      await userAPages.conversationList().openConversation(channelName);
+      await userAComponents.inputBarControls().clickShareImage(imageFilePath);
+
+      expect(await userBPages.conversation().isImageFromUserVisible(userA)).toBeTruthy();
+    });
+
+    await test.step('User B can open the image preview and see the image', async () => {
+      // Click on the image to open it in a preview
+      await userBPages.conversation().clickImage(userA);
+
+      // Verify that the detail view modal is visible
+      expect(await userBModals.detailViewModal().isVisible()).toBeTruthy();
+      expect(await userBModals.detailViewModal().isImageVisible()).toBeTruthy();
+    });
+
+    await test.step('User B can download the image', async () => {
+      // Click on the download button to download the image
+      const filePath = await userBModals.detailViewModal().downloadAsset();
+      const downloadQRCodeValue = await getLocalQRCodeValue(filePath);
+      const localQRCodeValue = await getLocalQRCodeValue(imageFilePath);
+      expect(downloadQRCodeValue).toBe(localQRCodeValue);
+    });
+
+    await test.step('User B reacts to A’s image', async () => {
+      await userBModals.detailViewModal().givePlusOneReaction();
+      await userBModals.detailViewModal().closeModal();
+      expect(await userBPages.conversation().isPlusOneReactionVisible()).toBeTruthy();
+    });
+
+    await test.step('User A can see the reaction', async () => {
+      // TODO: Bug [WPB-18226], remove this when fixed
+      await userAPageManager.refreshPage({waitUntil: 'load'});
+
+      expect(await userAPages.conversation().isPlusOneReactionVisible()).toBeTruthy();
+    });
+
+    await test.step('User A sends video message', async () => {
+      await userAComponents.inputBarControls().clickShareFile(videoFilePath);
+      expect(await userAPages.conversation().isVideoMessageVisible()).toBeTruthy();
+    });
+
+    await test.step('User B can play the message', async () => {
```

Comment:
I think You can't play the message :D You can play video :)

```suggestion
    await test.step('User B can play the received video', async () => {
```